### ISSUE TITLE
Crashing when loading games.(Nvidia Shield TV)

### ISSUE THREAD

**Issue by foxrumor** [hrydgard/ppsspp#11643]
I'm just got a brand new Nvidia Shield TV and PPSSPP is crashing immediately after clicking on a game.

I've tried the latest development build and the problem still exists. I've also attempted all the settings in the PPSSPP is crashing guide, but that didn't fix it. I also read that someone else was having the same issue and it was marked as solved. That doesn't seem to be true. I hope something can be done about this.

**Comment by hrydgard** [comment]
That's weird, last I tried it, it worked just fine.

If you can access the file system, you might want to try to go to PSP/SYSTEM/ppsspp.ini (if it exists) and change GPUBackend to 0. Does that help?

**Comment by foxrumor** [comment]
Changing the GraphicsBackend to 0 does seem to have fixed it. Thanks for the help.

**Comment by foxrumor** [comment]
Aditionally, I was able to go back in and change the settings to Vulcan and everything worked wonderfully.

**Comment by unknownbrackets** [comment]
If you rename ppsspp.ini to backup.ini, does it start crashing again?

If yes, delete ppsspp.ini and rename backup.ini back.

-[Unknown]

**Comment by hrydgard** [comment]
Failed to reproduce this on a fresh build, even deleting the PSP folder entirely and reinstallling the app. Closing.

**Comment by bettik** [comment]
The Shield TV (2017 model) is now updated to "Android Experience 8" which runs Android Pie (9).  PPSSPP is crashing when loading any game for me now.  I've tried OpenGL and Vulkan.  Fresh installs of 1.8.0 from play store as well as latest nightly.  I've tried the suggestions regarding ppsspp.ini, as well.  All with no change.

Logcat attached.
[logcat.txt](https://github.com/hrydgard/ppsspp/files/3511594/logcat.txt)
